ci_cd/tasks/update_deps.py

```diff
@@ -285,7 +297,7 @@ def update_deps(  # pylint: disable=too-many-branches,too-many-locals,too-many-s
             error = True
             continue
 
-        latest_version: str = match.group("version")
```

Comment:
Converting the matched version string to a Version object is a good use of the packaging library's capabilities. It allows for more sophisticated handling of version strings. However, there is no error handling if the Version constructor fails (e.g., due to an invalid version string).